src/core/EdFi.DataManagementService.Core/Response/FailureResponse.cs

```diff
@@ -8,7 +8,7 @@ namespace EdFi.DataManagementService.Core.Response;
 /// <summary>
 /// Provides human-readable summary of the failure responses
 /// </summary>
-internal record FailureResponse(
+public record FailureResponse(
```

Comment:
Does this need to be `public` instead of `internal`? I think Brad just changed this to `internal` in the past day or so.